composables/price.ts

```diff
@@ -52,13 +52,24 @@ const aggregatorV3InterfaceABI = [
 
 // Mainnet ETH/USD Chainlink contract
 const ethUsdChainlinkAddress = '0x5f4ec3df9cbd43714fe2740f5e3616155c5b8419'
+// Polygon ETH/USD Chainlink contract
+const polygonEthUsdAddress = '0x7bac85a8a13a4bcd8abb3eb7d6b4d632c5a57676'
 
 let ethUsdChainlinkContract: ethers.Contract
+let polygonEthUsdChainlinkContract: ethers.Contract
 
 export const useEthPriceFeed = async () => {
+  const provider = await useRpcProvider()
   if (!ethUsdChainlinkContract) {
-    const provider = await useRpcProvider()
```

Comment:
Getting the provider should be inside the "if" clause bc it is only needed if the property doesn't exist yet, but I'll fix it after merge.

Reply:
No probs ✌🏽